library/solr

```diff
@@ -107,21 +117,21 @@ GitCommit: 86159575c4b6da2010ae9ee003bf5a6de31157cb
 Directory: 7.7/slim
 
 Tags: 6.6.6, 6.6, 6
-Architectures: amd64
+Architectures: amd64, arm64v8
 GitCommit: 86159575c4b6da2010ae9ee003bf5a6de31157cb
 Directory: 6.6
 
 Tags: 6.6.6-slim, 6.6-slim, 6-slim
-Architectures: amd64
+Architectures: amd64, arm64v8
 GitCommit: 86159575c4b6da2010ae9ee003bf5a6de31157cb
 Directory: 6.6/slim
 
 Tags: 5.5.5, 5.5, 5
-Architectures: amd64
+Architectures: amd64, arm64v8
 GitCommit: 86159575c4b6da2010ae9ee003bf5a6de31157cb
 Directory: 5.5
 
 Tags: 5.5.5-slim, 5.5-slim, 5-slim
-Architectures: amd64
+Architectures: amd64, arm64v8
```

Comment:
@dsmiley, `arm64v8` was removed for `6.x` and `5.x` in https://github.com/docker-library/official-images/pull/5973.  Is it correct to add them back?

Reply:
Great attention to detail there!  Let's not add them back.   @gadilashashank how did these get added?  I suspect it was a script.

Reply:
@dsmiley I simply followed the instructions at [https://github.com/docker-solr/docker-solr/blob/master/update.md#update-the-official-images-repository](https://github.com/docker-solr/docker-solr/blob/master/update.md#update-the-official-images-repository) so I believe the `generate-stackbrew-library.sh` needs to be updated. For now I'll manually remove `arm64v8` for `6.x` and `5.x`

Reply:
@dsmiley @yosifkit I've manually amended the commit this once. Please review and let me know if this looks good. I'll open an issue on the downstream repo for this.